test/unit/test_data_io.py

```diff
@@ -383,12 +383,16 @@ def test_get_training_data_iters():
         vcb = vocab.build_from_paths([data['source'], data['target']])
 
         train_iter, val_iter, config_data = data_io.get_training_data_iters(data['source'], data['target'],
+                                                                            [],
```

Comment:
I think this calls for keyword argument calling of this function  :)

Reply:
Done (not pushed yet, though).